osmaxx-py/osmaxx/tests/test_utils.py

```diff
@@ -0,0 +1,31 @@
+import string
+import random
+from django.test import TestCase
+import os
```

Comment:
Don't we put standard library imports (`os`) before framework library imports (`django.test`)?